USca/USca-Server/Shared/ServerDbContext.cs

```diff
@@ -19,17 +19,23 @@ public ServerDbContext()
                 _created = true;
                 Database.EnsureDeleted();
                 Database.EnsureCreated();
-            }
-
-            if (Users != null)
-            {
-                Users.Add(new() { Name = "Bob", Surname = "Jones", Username = "user1", Password = "1234" });
-                Users.Add(new() { Name = "Bab", Surname = "Janes", Username = "user2", Password = "1234" });
-                Users.Add(new() { Name = "Bib", Surname = "Jines", Username = "user3", Password = "1234" });
+                LoadInitialData();
             }
             SaveChanges();
         }
 
+        private void LoadInitialData()
+        {
+            Users.Add(new() { Name = "Bob", Surname = "Jones", Username = "user1", Password = "1234" });
+            Users.Add(new() { Name = "Bab", Surname = "Janes", Username = "user2", Password = "1234" });
+            Users.Add(new() { Name = "Bib", Surname = "Jines", Username = "user3", Password = "1234" });
+
+            Tags.Add(new() { Address = 1, Name = "Tank01" });
```

Comment:
Technically the tags we have in the RTU right now are analog, but `Tag` defaults to digital.

Reply:
https://github.com/magley/untitled-scada-project/pull/20/commits/4d70e46afae73ec725d2f254335b4eff2f87a53f